src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -7,58 +7,79 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:viewModel="using:Microsoft.PowerToys.Settings.UI.ViewModels"
     xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    xmlns:Lib="using:Microsoft.PowerToys.Settings.UI.Lib"
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Page.Resources>
-        <viewModel:RemapKeysModel x:Key="dummyData"/>
         <viewModel:KeyboardManagerViewModel x:Key="eventViewModel"/>
 
-        <DataTemplate x:Name="KeysListViewTemplate" x:DataType="viewModel:RemapKeysModel">
-            <Grid Height="56">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="80"/>
-                    <ColumnDefinition Width="80"/>
-                    <ColumnDefinition Width="120"/>
-                </Grid.ColumnDefinitions>
-                <Border 
-                        Background="{ThemeResource SystemBaseLowColor}"
-                        CornerRadius="4"
-                        Grid.Column="0"
-                        Padding="14,0,14,0"
-                        Margin="0,0,0,0"
-                        Height="36"
-                        VerticalAlignment="Center"
-                        HorizontalAlignment="Left">
-                    <TextBlock FontWeight="SemiBold"
-                               VerticalAlignment="Center"
-                               TextAlignment="Center"
-                                FontSize="12"
-                                Text="{Binding From}">
-                    </TextBlock>
-                </Border>
+        <DataTemplate x:Name="KeysListViewTemplate" x:DataType="Lib:KeysDataModel">
```

Comment:
![image](https://user-images.githubusercontent.com/32061677/79937303-5a713a80-840e-11ea-965c-ef3865c4edc4.png)
Is there any way we can align the keys and the arrows when some keys are different sized? Maybe something like a grid?
Similar issue exists for shortcuts as well, but it might be harder to align in this case? In #2299 we are making the Edit UI aligned, but in that case the drop downs are fixed width so it's easier to handle, whereas here it seems to be text.
![image](https://user-images.githubusercontent.com/32061677/79937403-9b694f00-840e-11ea-816a-d40927621e59.png)

Reply:
I personally prefer it this way, but maybe it can be changed. @saahmedm what do you think?